### ISSUE TITLE
Consider adding charts to literature resource search

### ISSUE THREAD

**Issue by nickynicolson** [gbif/portal-feedback#5035]
Organisations often need to report on citations made on the data that they mobilise through GBIF, and some simple chart options (as given in the occurrence part of the web portal) would be helpful to show trends in citation over time

**Comment by ManonGros** [comment]
Hi @nickynicolson there are a few things already available. For example the chart of citations over time on the metrics tab of publisher pages. See for example: https://www.gbif.org/publisher/27b44945-6a9a-4d3d-a19e-81b5335592d2/metrics

There is also the possibility to download aggregated download activity over time. See the download activity button on the publisher page: https://www.gbif.org/publisher/27b44945-6a9a-4d3d-a19e-81b5335592d2.

What kind of charts would you like to see?

**Comment by nickynicolson** [comment]
I'm correct that downloads differ from citations?
We need citations summarised by year & quarter, the summarisation options (facets / charts) on the portal currently appear to only allow us to summarise by year.

**Comment by ManonGros** [comment]
Yes downloads and citations are two different metrics. Citation summarised by quarter would make sense. I will leave the issue open so the developers can take a look.

You might already know this but you can download the citations for a given publisher as a TSV file: https://www.gbif.org/resource/search?contentType=literature&publishingOrganizationKey=27b44945-6a9a-4d3d-a19e-81b5335592d2 (see the "download as TSV" button in the upper right corner).

**Comment by MortenHofft** [comment]
What kind of summaries are you interested in? just counts per quarter? Or breakdowns of some other dimension pr quarter?

We have a bit more on the hosted portals
https://hp-theme.gbif-staging.org/publisher/27b44945-6a9a-4d3d-a19e-81b5335592d2/citations
scroll below table
